build.gradle.kts

```diff
@@ -109,4 +110,28 @@ publishing {
             artifact(tasks.getByName("javadocJar"))
```

Comment:
We need to add this line in new publications to bintray, because they are in infra.publishing